aws-rds-eventsubscription/src/main/java/software/amazon/rds/eventsubscription/UpdateHandler.java

```diff
@@ -11,44 +12,58 @@
 import software.amazon.cloudformation.proxy.ProgressEvent;
 import software.amazon.cloudformation.proxy.ProxyClient;
 import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+import software.amazon.rds.common.handler.Commons;
+import software.amazon.rds.common.handler.Tagging;
 
 public class UpdateHandler extends BaseHandlerStd {
     protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
-      final AmazonWebServicesClientProxy proxy,
-      final ResourceHandlerRequest<ResourceModel> request,
-      final CallbackContext callbackContext,
-      final ProxyClient<RdsClient> proxyClient,
-      final Logger logger) {
-      final ResourceModel model = request.getDesiredResourceState();
-      final ResourceModel previousModel = request.getPreviousResourceState();
-      final Set<String> currentSourceIds = Optional.ofNullable(model.getSourceIds()).orElse(Collections.emptySet());
-      final Set<String> previousSourceIds = Optional.ofNullable(previousModel.getSourceIds()).orElse(Collections.emptySet());
+            final AmazonWebServicesClientProxy proxy,
+            final ResourceHandlerRequest<ResourceModel> request,
+            final CallbackContext callbackContext,
+            final ProxyClient<RdsClient> proxyClient,
+            final Logger logger) {
+        final ResourceModel model = request.getDesiredResourceState();
```

Comment:
nit: try to keep the naming schema consistent: the tags below use `previous` and `desired` prefixes, but the model objects don't